Source/DiabloUI/credits.cpp

```diff
@@ -94,8 +94,9 @@ void CreditsRenderer::Render()
 	prev_offset_y_ = offsetY;
 
 	SDL_FillRect(DiabloUiSurface(), nullptr, 0x000000);
-	DrawArt({ PANEL_LEFT - 320, UI_OFFSET_Y }, &ArtBackgroundWidescreen);
-	DrawArt({ PANEL_LEFT, UI_OFFSET_Y }, &ArtBackground);
+	auto &uiPosition = GetUIRectangle().position;

```

Comment:
It's small and shouldn't be changed.
```suggestion
	const Point uiPosition = GetUIRectangle().position;
```

Reply:
Changed it. Here and in other occurrences (point and rectangle and for mainpanel), 🙂